libp2p/security/security_multistream.py

```diff
@@ -24,64 +21,48 @@
 
 
 class SecurityMultistream(ABC):
-    transports: Dict[TProtocol, "ISecureTransport"]
-    multiselect: "Multiselect"
-    multiselect_client: "MultiselectClient"
+    transports: Dict[TProtocol, ISecureTransport]
+    multiselect: Multiselect
+    multiselect_client: MultiselectClient
 
-    def __init__(self) -> None:
-        # Map protocol to secure transport
+    def __init__(self, secure_transports_by_protocol) -> None:
```

Comment:
missing type hints?  Any chance we can get things typed and then enforced via CI?

Reply:
yep, there is progress on restoring use of `mypy` in this repo -- i think it has already been completed so I'll have to fix this before we can merge this in.

Reply:
this has been addressed in #250.